jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -197,7 +197,8 @@ git --no-pager diff --name-only HEAD \$BASE -- ${PREMERGE_DOCKERFILE} || true"""
                                               execPattern           : '**/target/jacoco.exec',
                                               classPattern          : 'target/jacoco_classes/',
                                               sourceInclusionPattern: '**/*.java,**/*.scala',
-                                              sourcePattern         : sourcePattern
+                                              sourcePattern         : sourcePattern,
+                                              exclude               : '*/META-INF/versions/*',
```

Comment:
the `exclude` param should be `exclusionPattern`, let me double conform

Reply:
@liurenjie1024  
`exclusionPattern : 'target/jacoco_classes/*/META-INF/versions/*',`  does not work here, 
Looks like [Jacoco plugin] run below steps

1, firstly tries to gather all classes together;

2 ,then it excludes classes defined by exclusionPattern

The first step fails due to the dup ArraysShim.class

I guess this is why instead of with param `exclusionPattern` for [Jacoco Plugin],
we remove folders in advance here:  https://github.com/NVIDIA/spark-rapids/blob/branch-24.08/jenkins/spark-premerge-build.sh#L95-L96


BTW, the pre-merge CI failed, because the path of ArraysShim.class in sparkxxx/MATE-INFO has been changed with this PR:

`spark320/META-INF/versions/11/org/roaringbitmap/    
 -- >      /META-INF/versions/11/com/nvidia/shaded/spark/org/roaringbitmap/`

So please help to update the line?

https://github.com/NVIDIA/spark-rapids/blob/branch-24.08/jenkins/spark-premerge-build.sh#L95

```
rm -rf com/nvidia/shaded/ org/openucx/          spark${SPK_VER}/META-INF/versions/*           spark-shared/com/nvidia/spark/udf/ spark${SPK_VER}/com/nvidia/spark/udf/

```